AppCenter/AppCenter/MSAppCenter.m

```diff
@@ -214,6 +214,7 @@ - (BOOL)configureWithSecretString:(NSString *)secretString fromApplication:(BOOL
       }
       if (!self.defaultTransmissionTargetToken) {
         self.defaultTransmissionTargetToken = [MSUtility transmissionTargetTokenFrom:secretString];
+        NSLog(@"TTT: %@", self.defaultTransmissionTargetToken);
```

Comment:
This shouldn't be `NSLog(..)` but I thinnk it's just ann oversight from testing?

Reply:
Yup, needs to be removed